packages/web-components/fast-element/src/template.ts

```diff
@@ -139,7 +169,7 @@ export class ViewTemplate<TSource = any, TParent = any> extends Directive
 }
 
 // Much thanks to LitHTML for working this out!
-export const lastAttributeNameRegex =
```

Comment:
Wouldn't this be a breaking change since this is being exported from the `index.ts`?

Reply:
Good catch. Technically, yes. That was never intended to be exported though. I can't imagine anyone using that directly.